docs/en/Configuring-Alluxio-with-NFS.md

```diff
@@ -32,18 +32,17 @@ directory `/mnt/nfs`, the following environment variable assignment needs to be
 `conf/alluxio-site.properties`:
 
 ```
-properties
+alluxio.master.hostname=localhost
 alluxio.underfs.address=/mnt/nfs
 ```
 
 ## Running Alluxio with NFS
 
-After everything is configured, you can start up Alluxio locally to see that everything works.
+Simply run the following command to start Alluxio filesystem.
 
 {% include Configuring-Alluxio-with-NFS/start-alluxio.md %}
 
-This should start an Alluxio master and an Alluxio worker. You can see the master UI at
-[http://localhost:19999](http://localhost:19999).
+To verify that Alluxio is running, you can visit http://localhost:19999, or see the log in the logs folder.
```

Comment:
is removing the link intentional?

Reply:
Updated, thanks for pointing out the mistake.